core/app/models/spree/promotion/rules/nth_order.rb

```diff
@@ -0,0 +1,43 @@
+module Spree
+  class Promotion
+    module Rules
+      class NthOrder < PromotionRule
+        preference :nth_order, :integer, default: 2
+        # It does not make sense to have this apply to the first order using preferred_nth_order == 1
+        # Instead we could use the first_order rule
```

Comment:
Yeah it just seems like a weird restriction from a user point-of-view to me.

Alright: I'm :+1: on this and happy to merge it as is. But for what it's worth, I'd actually want to get rid of the first order promotion. I think doing this without requiring user sign-in is almost a little silly particularly in times where I can just add a +1, +2, +3 to my email address to keep using a first order promotion. And outside of that difference the two promotion rules behave the same, so it seems simpler and more logical to have just one.

But again: happy to follow the majority here, just wanted to chime in with my two cents.